apache/docker-entrypoint.sh

```diff
@@ -3,7 +3,30 @@
 
 # PWD=`pwd`
 
+run_entrypoint_scripts() {
+    phase="$1"
+    shift
+    shopt -s nullglob
+    for file in /entrypoint-scripts/$phase-setup/*.sh; do
+        if test ! -f "$file"; then
+            echo "$phase-setup/$file is not a regular file, ignoring."
+        fi
+        echo "Running $phase-setup script $file:"
+        bash "$file" "$@"
+        # Exit in case of an error in a script.
+        exit_code=$?
+        if test $exit_code -ne 0; then
+            echo 'Error in script, exiting!'
+            exit $exit_code
+        fi
+        echo 'Done.'
+    done
+    shopt -u nullglob
+}
+
 if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
+  run_entrypoint_scripts pre $@
```

Comment:
this re-introduces the issue of too-eagerly splitting arguments containing whitespace.